pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -17745,6 +17745,18 @@ final: prev:
     meta.homepage = "https://github.com/samodostal/image.nvim/";
   };
 
+  resession-nvim = buildVimPlugin {
+    pname = "resession-nvim";
```

Comment:
```suggestion
    pname = "resession.nvim";
```

Reply:
Hmm, I thought these were supposed to have hyphens instead of dots?